modules/test/playwright/pages/users-admin-web/UsersAndOrganizationsPage.ts

```diff
@@ -468,6 +508,20 @@ export class UsersAndOrganizationsPage {
 		]);
 	}
 
+	async goToOrganizationsWithLimitedAccess() {
+		await this.applicationsMenuPage.goToUsersAndOrganizationsLimitedAccess();
```

Comment:
can we use the same "format"?
`goToOrganizationsWithLimitedAccess` ---> here we have With
`goToUsersAndOrganizationsLimitedAccess` ---> here we don't have With
personally I prefer using "With"